core/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -434,16 +436,6 @@ private long sizeInBytesByMinGen(long minGeneration) {
     }
 
     /**
-     * Returns the size in bytes of the translog files with ops above the given seqNo
-     */
-    private long sizeOfGensAboveSeqNoInBytes(long minSeqNo) {
```

Comment:
Why are we removing this method here?

Reply:
I removed this method, because it isn't used. Do you want me to remove it outside of this PR on master/6.x branches?

Reply:
Yes.

Reply:
Ok, I've brought the method back in this PR.